libraries/Microsoft.Bot.Connector/Authentication/MicrosoftAppCredentials.cs

```diff
@@ -100,12 +88,6 @@ public MicrosoftAppCredentials(string appId, string password)
         /// </summary>
         public virtual string OAuthScope { get { return AuthenticationConstants.ToChannelFromBotOAuthScope; } }
 
```

Comment:
Removing a public property is technically a breaking change. Probably need to mark as obsolete, explaining why it's gone and how to accomplish this now and then hide it with editor visible attribute so it doesn't show up in intellisense for new code, but doesn't break old code that might have said it.

Reply:
Good point, thanks!